mllib/src/test/scala/org/apache/spark/ml/tree/impl/RandomForestSuite.scala

```diff
@@ -509,6 +510,25 @@ class RandomForestSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(mapToVec(map.toMap) ~== mapToVec(expected) relTol 0.01)
   }
 
+  test("options for feature subset size in RandomForest - SPARK-3724") {
```

Comment:
There are already tests for feature subset strategy on L421. You can just add to those instead of creating this new test.